includes/Frontend/Payment.php

```
@@ -161,208 +161,173 @@ public function payment_page( $content ) {
+                            'id'             => $pack_id,
+                            'title'          => $order_title,
+                            'subtotal'       => $pack_cost,
+                            'billing_amount' => $billing_amount,
+                            'recurring_pay'  => isset( $pack->meta_value['recurring_pay'] ) ? $pack->meta_value['recurring_pay'] : 'no',
+                        ] );
+                    } elseif ( $post_id ) {
+                        $form              = new Admin\Forms\Form(
+                            get_post_meta( $post_id, '_wpuf_form_id', true )
+                        );
+                        $force_pack        = $form->is_enabled_force_pack();
+                        $pay_per_post      = $form->is_enabled_pay_per_post();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$pay_per_post'. (undefined)

(UnusedLocalVariable)